include/avif/avif.h

```diff
@@ -539,6 +539,7 @@ typedef struct avifImage
     avifRWData xmp;
 } avifImage;
 
+// If avifImageCreate() or avifImageCreateEmpty() returns NULL, an AVIF_RESULT_OUT_OF_MEMORY error happened.
```

Comment:
1. Similar to my comment in CHANGELOG.md, in comments like this one, it would be clearer to say "an out-of-memory error happened" or "memory allocation failed".

2. Since `avifImageCreate()` has input parameters, it could fail because of invalid arguments. So it would be good to make it clearer that out-of-memory error is not the only reason for `avifImageCreate()` failure. Alternatively, we can document that `avifImageCreate()` faithfully sets the width, height, depth, and yuvFormat members of the new image to the input parameters, and this will become an API contract.